public/locales/fr/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "settingsFor": "Réglages pour",
+  "saveSettings": "Sauvegarder les réglages",
```

Comment:
May we make this one shorter so it fits better in the button? It may be problematic on mobile.
```suggestion
  "saveSettings": "Sauvegarder",
```